assets/src/js/admin/onboarding-wizard/app/steps/location/index.js

```diff
@@ -20,12 +20,17 @@ const Location = () => {
 	const state = configuration.state;
 	const currency = configuration.currency;
 
+	const onChangeCountry = ( value ) => {
+		dispatch( fetchStateList( value, dispatch ) );
```

Comment:
@henryholtgeerts here is how I solved the fetch/set for the States List. Changing the country value dispatches two separate actions. The `fetchStateList` action is passed `dispatch` as a callback because you can't call `useStoreValue()` within React Hooks, or something like that.